# Pensamiento geométrico para visualizar y argumentar

# Clase 5

## Círculos II

### Longitud de cuerda

<img src="cuerda.png" align="middle" width="300">

$$ \ell=2R\sin\left(\dfrac{\theta}{2}\right) $$

$$ \ell=\sqrt{2R^2(1-\cos\theta)} $$

$$ \ell=2R\sin\left(\dfrac{\pi t}{360}\right) $$

$$ \ell=\sqrt{2R^2\left[1-\cos\left(\dfrac{\pi t}{180}\right)\right]} $$

* $\ell$ : longitud de cuerda
* $R$ : radio
* $\theta$ : ángulo (radianes)
* $t$ : ángulo (grados)

### Segmento circular

<img src="segmento.png" align="middle" width="300">

$$ A=\dfrac{R^2}{2}(\theta-\sin\theta) $$

$$ A=\dfrac{R^2}{2}\left[\dfrac{\pi t}{180}-\sin\left(\dfrac{\pi t}{180}\right)\right] $$

## Situación: El arte Burkina

**En un pueblo africano llamado Tiebele en Burkina Faso la corte real se aloja en pequeñas estructuras de adobe (arcilla y arena), que están decoradas con pinturas naturales que forman patrones geométricos. El propósito, además de estético, es para diferenciarlas de las casas de la gente común.**

**Tlahuilmalitzin es una artesana mexicana la cual admira el arte de cualquier región. Y quiere pintar una pared con un diseño parecido al de Burkina Faso. Pero ha notado que el tipo de figuras que adornan las paredes en Burkina son sencillas. Lo único que tiene que hacer es decidir su diseño de acuerdo con su presupuesto.**

<img src="extra1-1.png" align="middle" width="300">

**1. Dado que los anillos están separados a la misma distancia, Tlahuilmalitzin se pregunta si el área gris que se muestra en la siguiente figura, es: [1 punto]**

a)  $\dfrac{1}{2}$ de área del círculo mayor

b)  $\dfrac{3}{4}$ de área del círculo mayor

c)  $\dfrac{3}{8}$ de área del círculo mayor

d)  $\dfrac{5}{8}$ de área del círculo mayor

$ R_1 $ para el primer círculo

$ R_2=2R_1 $ para el segundo círculo

$ R_3=3R_1 $ para el tercer círculo

$ R_4=4R_1 $ para el tercer círculo

$$ A_1=\pi {R_1}^2 $$

$$ A_2=\pi {R_2}^2=\pi(2R_1)^2=\pi(4{R_1}^2)=4\pi {R_1}^2=4A_1 $$

$$ A_3=\pi {R_3}^2=\pi(3R_1)^2=\pi(9{R_1}^2)=9\pi {R_1}^2=9A_1 $$

$$ A_4=\pi{R_4}^2=\pi(4R_1)^2=16\pi{R_1}^2=16A_1 $$

$$ A_S=A_4-A_3+A_2-A_1 $$

$$ A_S=(16A_1)-(9A_1)+(4A_1)-A_1=10A_1 $$

Vamos a comparar el área sombreada con el área del circulo mayor:

$$ \dfrac{A_S}{A_4}=\dfrac{10A_1}{16A_1}=\dfrac{5}{8} $$

a)  $\dfrac{1}{2}$ de área del círculo mayor

b)  $\dfrac{3}{4}$ de área del círculo mayor

c)  $\dfrac{3}{8}$ de área del círculo mayor

d)  $\dfrac{5}{8}$ de área del círculo mayor

**Solución:** d)

**2. Tlahuilmalitzin hizo otro diseño con las mismas circunferencias del modelo anterior, donde el punto $A$ está sobre la bisectriz del $\angle CBE$ , si $BC$ mide $1\ \rm{dm}$, el área de la estrella es: [2 puntos]**

<img src="extra1-2.png" align="middle" width="300">

Triángulo $ABC$:

* Datos:
    * $\overline{AB}=\dfrac{1}{2}\ dm$
    * $\overline{BC}=1\ dm$
    * $\angle ABC=45^\circ$
    * $A=?$

* Fórmula:

<img src="triangulo.png" align="middle" width="300">

*"Para cualquier triángulo el área es la mitad del producto de dos catetos y el seno del ángulo entre estos dos catetos."*

$$ \text{Area}=\dfrac{ab\sin C}{2}=\dfrac{bc\sin A}{2}=\dfrac{ac\sin B}{2} $$

$$ A_\Delta=\dfrac{\left(\dfrac{1}{2}dm\right)(1dm)\sin{45^\circ}}{2} $$

In [1]:
((1/2)*(1)*sind(45))/2

0.1767766952966369

$$ =\dfrac{\left(\dfrac{1}{2}dm\right)(1dm)\dfrac{\sqrt{2}}{2}}{2}=\dfrac{\dfrac{\sqrt{2}}{4}}{2} dm^2=\dfrac{\sqrt{2}}{8} dm^2 $$

$$ A_S=\sqrt{2}dm^2 $$

In [2]:
8*0.1767766952966369

1.4142135623730951

a)  $\dfrac{\sqrt{2}}{2}\ \rm{dm}^2$

b)  $\sqrt{2}\ \rm{dm}^2$

c)  $2\sqrt{2}\ \rm{dm}^2$

d)  $\dfrac{\sqrt{2}}{4}\ \rm{dm}^2$

**Solución:** b)

**3. Tlahuilmalitzin se pregunta ¿cómo son las áreas grises de ambas figuras? [2 puntos]**

<img src="extra1-3.png" align="middle" width="300">

$$ A_1=\pi {R_1}^2=\pi\left(0.025\ m\right)^2=\dfrac{\pi}{1600}\ m^2 $$

$$ A_\text{anillos}=10A_1=\frac{\pi}{160}\ m^2 $$

In [5]:
5π/8

1.9634954084936207

$A_\text{estrella}=\sqrt{2}=1.4142$

$A_\text{anillos}=1.9635$

a)  El área de los anillos es la mitad que el área de la estrella.

b)  El área de los anillos es mayor que la mitad del área de la estrella.

c)  El área de los anillos es igual que el área de la estrella.

d)  El área de los anillos es menor que la mitad del área de la estrella.

**Solución:** b) 

**4. Tlahuilmalitzin quiere adornar las paredes exteriores de su casa que construirá en Cuernavaca al estilo de las casas de Burkina Faso, ella tiene dos diseños combinando el de los anillos y las estrellas. ¿Cuál es el área en $\rm{m}^2$ que ocupan todos los anillos de ambos diseños? [3 puntos]**

<img src="extra1-4.png" align="middle" width="500">

<img src="extra1-5.png" align="middle" width="500">

$$ A_1=\pi {R_1}^2=\pi\left(0.025\ m\right)^2=\dfrac{\pi}{1600}\ m^2 $$

$$ A_\text{anillos}=(16A_1)-(9A_1)+(4A_1)-A_1=10A_1=\frac{\pi}{160}\ m^2 $$

El área total debida a los anillos es

$$ 20A_\text{anillos}=\dfrac{20}{160}\pi\ m^2=\dfrac{2}{16}\pi\ m^2=\dfrac{1}{8}\pi\ m^2 $$

a)  $\dfrac{5\pi}{4}\ \rm{m}^2$

b)  $\dfrac{13\pi}{80}\ \rm{m}^2$

c)  $\dfrac{13\pi}{5}\ \rm{m}^2$

d)  $\dfrac{\pi}{8}\ \rm{m}^2$

**Solución:** d)

## Situación: Satélites artificiales

**Un satélite artificial es un ingenio enviado en una lanzadera espacial que se mantiene en órbita alrededor de cuerpos del espacio. Los satélites artificiales orbitan alrededor de satélites naturales, asteroides o planetas. Tras su vida útil, pueden quedar orbitando como basura espacial, o pueden desintegrarse reingresando en la atmósfera (cosa que ocurre solamente si su órbita es de poca altura).**

**Fuente: https://es.wikipedia.org/wiki/Sat%C3%A9lite_artificial**

a)  $\dfrac{40000}{\pi}$

b)  $\dfrac{20000}{\pi}$

c)  $40000\pi$

d)  $20000\pi$

**2. Si un satélite orbita circularmente a $2000\ \rm{km}$ de la superficie terrestre, ¿cuántos kilómetros mide la circunferencia de su órbita? [2 puntos]**


a)  $40000+4000\pi$

b)  $40000-4000\pi$

c)  $\dfrac{20000+2000\pi}{\pi}$

d)  $\dfrac{20000-2000\pi}{\pi}$

***Solución:***


**3. Si cada día un satélite recorre un ángulo de $\dfrac{\pi}{12}\ \rm{rad}$ , ¿cuántos días transcurren para que de una vuelta completa a la Tierra? [2 puntos]**

a)  $24$

b)  $12$

c)  $\dfrac{1}{24}$

d)  $\dfrac{1}{12}$

***Solución:***


**4.  Se planea lanzar un satélite a una altura de $1,000\ \rm{km}$ sobre la superficie terrestre, si este da una vuelta a la Tierra en $10$ días, ¿cuántos kilómetros sobre la superficie terrestre recorrerá cada día el satélite? [3 puntos]**

a)  $4000+200\pi$

b)  $200\pi$

c)  $\dfrac{720000+36000\pi}{\pi}$

d)  $36000$

***Solución:***


## Situación: Funámbulo

Un funámbulo o funambulista es un equilibrista que camina sobre un alambre o cuerda tensada.

Paul, que es uno de estos acróbatas, realiza un espectáculo de equilibrio a una altura de $5$ metros sobre el nivel del suelo, caminando sobre un cable de acero tenso entre los puntos $B$ y $C$ de las paredes de dos edificios separadas por $10$ metros, como se puede ver en la figura. Mientras camina del punto $B$ al punto $C$, lo sigue un reflector que se encuentra a $15$ metros de altura, en el punto $R$ de la figura, que provoca que genere una sombra en el piso.

#### Ojo: Las figuras bonitas no vienen en el extra. Yo las hice para que entendieran mejor el problema. Eso deben hacer cuando se enfrenten a un problema.

<img src="extra4-1.png" align="middle" width="200">

<img src="extra_sol1.png" align="middle" width="200">

**1. Cuando Paul ha recorrido la mitad del cable, la distancia $\overline{AS}$ que ha recorrido su sombra, redondeada a una décima, es**

<img src="extra_sol2.png" align="middle" width="200">

Semejante: $\Delta RAS$ y $\Delta RBP$

$$ \dfrac{15}{20}=\dfrac{5}{x} $$

$$ \dfrac{x}{5}=\dfrac{20}{15} $$

$$ x=\dfrac{20}{15}5 $$

In [6]:
(20/15)*5

6.666666666666666

La longitud del segmente $\overline{AS}$, redondeada a una décima, es 6.7 m

**2. Cuando el punto sombra $S$ del piso, alcanza la base del edificio en el punto $D$, la distancia que Paul ha recorrido desde el punto $B$ es**

<img src="extra_sol3.png" align="middle" width="200">

**3. Cuando el punto sombra $S$ del piso llega al punto $D$, en ese instante se apaga el proyector que está en el punto $R$ y se prende otro proyector en el punto $A$ del piso y también provoca el punto $P$ de apoyo de Paul un punto sombra $S^{\prime}$, pero ahora sobre la pared $DG$ ¿A qué altura de la pared $DG$ (medida desde el piso), se encuentra el punto sombra en el instante en que se prende el proyector que se encuentra en $A$?**

<img src="extra_sol4.png" align="middle" width="300">

## Situación: Estrellas

**1. El lado de un triángulo equilátero mide $x$ unidades. Su área, en unidades cuadradas, es (2 puntos)**

<img src="equi.png" align="middle" width="200">

a)  $\dfrac{3x^2}{2}$

b)  $\dfrac{\sqrt{3}x^2}{4}$

c)  $\dfrac{x^2}{2}$

d)  $\dfrac{\sqrt{3}x^2}{2}$

***Solución:***

<img src="extra2-1.png" align="middle" width="200">

a)  $\dfrac{1}{3}$

b)  $\dfrac{1}{8}$

c)  $\dfrac{1}{4}$

d)  $\dfrac{1}{9}$

***Solución:***

**3. En la siguiente figura se muestra un segmento $AB$, cuya longitud vale $a$ unidades. El área sombreada, en unidades cuadradas, es (2 puntos)**

<img src="extra2-2.png" align="middle" width="200">

a)  $\dfrac{\sqrt{3}a^2}{3}$

b)  $\dfrac{2\sqrt{3}a^2}{3}$

c)  $\dfrac{\sqrt{3}a^2}{4}$

d)  $\dfrac{3\sqrt{3}a^2}{4}$

***Solución:***

**4. El segmento $AB$ de la siguiente figura tiene una longitud $a$. Entonces el área de la figura sombreada es (2 puntos)**

<img src="extra2-3.png" align="middle" width="200">

a)  $\dfrac{10\sqrt{3}a^2}{27}$

b)  $\dfrac{14\sqrt{3}a^2}{27}$

c)  $\dfrac{5\sqrt{3}a^2}{9}$

d)  $\dfrac{7\sqrt{3}a^2}{9}$

***Solución:***